plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -100,13 +103,16 @@ func (p *pusher) AddEventNonBlocking(e logs.LogEvent) {
 		p.startNonBlockCh <- struct{}{} // Unblock the select loop to recogonize the channel merge
 	})
 
+	util.GetLogBlocker().Add(int64(len(e.Message())))
+
 	// Drain the channel until new event can be added
 	for {
 		select {
 		case p.nonBlockingEventsCh <- e:
 			return
 		default:
 			<-p.nonBlockingEventsCh
+			util.GetLogBlocker().Subtract(int64(len(e.Message())))
```

Comment:
Instead of adding `int64(len(e.Message()))` everywhere, can't we just add a `Size()` to the `LogEvent` struct?

Reply:
Added Size()